geomesa-hbase/geomesa-hbase-datastore/src/test/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreTest.scala

```diff
@@ -151,6 +152,7 @@ class HBaseDataStoreTest extends Specification with LazyLogging {
         feature.getAttribute(attribute) mustEqual results.find(_.getID == feature.getID).get.getAttribute(attribute)
       }
     }
+    ds.getFeatureSource(typeName).getFeatures(query).size() mustEqual results.length
```

Comment:
yeah, it check the correct answers right above this - results is actually the expected results passed into the method, not the query results (confusingly named)